README.md

```diff
@@ -359,6 +373,7 @@ import TooltipOnElementComponent from 'ember-tooltips/components/tooltip-on-elem
 export default TooltipOnElementComponent.extend({
   effect: 'fade',
   side: 'bottom',
+  enableLazyRendering: true,
```

Comment:
No, I don't want to switch lazy rendering on by default until version 3.0.0.

I'm just showing it here to illustrate how to opt-in to this change for all of their tooltip/popover components.